src/main/java/com/stripe/model/Plan.java

```diff
@@ -1,218 +1,285 @@
-    return request(RequestMethod.POST, classUrl(Plan.class), params, Plan.class, options);
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/plans");
+    return request(ApiResource.RequestMethod.POST, url, params, Plan.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="delete">
-  /**
-   * Delete a plan.
-   */
-  public Plan delete() throws StripeException {
-    return delete((RequestOptions) null);
+  /** Retrieves the plan with the given ID. */
+  public static Plan retrieve(String plan) throws StripeException {
+    return retrieve(plan, (Map<String, Object>) null, (RequestOptions) null);
   }
 
-  /**
-   * Delete a plan.
-   */
-  public Plan delete(RequestOptions options) throws StripeException {
-    return request(RequestMethod.DELETE, instanceUrl(Plan.class, this.id), null, Plan.class,
-        options);
+  /** Retrieves the plan with the given ID. */
+  public static Plan retrieve(String plan, RequestOptions options) throws StripeException {
+    return retrieve(plan, (Map<String, Object>) null, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="list">
-  /**
-   * List all plans.
-   */
-  public static PlanCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+  /** Retrieves the plan with the given ID. */
+  public static Plan retrieve(String plan, Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url = String.format("%s%s", Stripe.getApiBase(), String.format("/v1/plans/%s", plan));
+    return request(ApiResource.RequestMethod.GET, url, params, Plan.class, options);
   }
 
   /**
-   * List all plans.
+   * Updates the specified plan by setting the values of the parameters passed. Any parameters not
+   * provided are left unchanged. By design, you cannot change a plan’s ID, amount, currency, or
+   * billing cycle.
    */
-  public static PlanCollection list(Map<String, Object> params, RequestOptions options)
-      throws StripeException {
-    return requestCollection(classUrl(Plan.class), params, PlanCollection.class, options);
+  public Plan update(Map<String, Object> params) throws StripeException {
+    return update(params, (RequestOptions) null);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="retrieve">
   /**
-   * Retrieve a plan.
+   * Updates the specified plan by setting the values of the parameters passed. Any parameters not
+   * provided are left unchanged. By design, you cannot change a plan’s ID, amount, currency, or
+   * billing cycle.
    */
-  public static Plan retrieve(String id) throws StripeException {
-    return retrieve(id, (RequestOptions) null);
+  public Plan update(Map<String, Object> params, RequestOptions options) throws StripeException {
+    String url =
+        String.format("%s%s", Stripe.getApiBase(), String.format("/v1/plans/%s", this.getId()));
+    return request(ApiResource.RequestMethod.POST, url, params, Plan.class, options);
```

Comment:
11) Update (standard): Instance method to update a resource, relying on ID attribute of the instance itself.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Plan.java#L223-L232